shared-libs/environment/test/index.spec.js

```diff
@@ -38,7 +39,7 @@ describe('environment', () => {
 
     const env = rewire('../src/index');
 
-    expect(env).to.deep.equal({
+    expect(env).to.deep.include({
```

Comment:
I think we should keep the strict `equal` test here, and be definitive of which properties should be exported, and we can use `chai-exclude` to exclude the functions.